### DISCUSSION TITLE
How can we interpret the difference in the width of uncertainty between report and infection curves for a same date range?

### DISCUSSION THREAD

**Discussion by avallecam** [epiforecasts/EpiNow2#635] [Q&A] [answered]
> Context: question made during one workshop in reaction to the output generated by this [tutorial section](https://epiverse-trace.github.io/tutorials-middle/create-forecast.html#create-a-short-term-forecast).

From the figure below we roughly identify two date ranges (labelled 1 and 2):

- In panel A, cases by date of report, the uncertainty of the mapped estimates between date range 1 and 2 is quite homogeneous, maybe slightly increasing in date range 2.
- In panel B, cases by date of infection, the difference in the uncertainty of the estimates between date range 1 and 2 is higher.
- In panel C, effective reproduction no., the difference in the uncertainty of the estimates resembles the difference between ranges 1 and 2 from panel B (which may be expected from how related they are in the renewal equation).

The question: 
- Is the observed uncertainty pattern in Panel A the result of a different calculation or a different set of underlying assumptions?

![image](https://github.com/epiforecasts/EpiNow2/assets/21230390/70e66681-b62c-42a2-a53c-009c73838394)

**Comment by sbfnk** [comment, marked as answer]
This is due to the delays - as reports are delayed from infections the expansion of uncertainty would happen later. Happy to hear suggestions for how this could be made clearer.

**Reply by avallecam** [reply]
Interesting observation. Actually, when comparing the outputs of without vs with `delays`, I've noticed the "displacement" of the infections curve days behind. Now I see that the expansion of the uncertainty in Panel B day range 2 is also due to the delays directly.

![image](https://github.com/epiforecasts/EpiNow2/assets/21230390/a1233062-767b-4798-b405-c1e9fe6e0257)

**Reply by avallecam** [reply]
but, looking back to Panel A, I get that due to delays this expansion of the uncertainty would happen later. It makes sense that Panel A, range 2, keeps the same homogeneous uncertainty as in Panel B, range 1. Thanks
